test/functional/legacy/105_filename_modifiers_spec.lua

```diff
@@ -45,6 +45,7 @@ describe('filename modifiers', function()
     execute([=[Put fnamemodify("abc\ndef",       ':S'      )]=])
     execute([=[set shell=tcsh]=])
     execute([=[Put fnamemodify("abc\ndef",       ':S'      )]=])
+    execute([=[Put expand('%:r:S') == shellescape(expand('%:r'))]=])
```

Comment:
`expand('%')` is the empty string when run in the lua test harness.  Something about that seems to break `expand('%:S')`.